catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/security/AccessControlPolicyExtension.java

```diff
@@ -76,8 +82,15 @@ public AccessControlPolicyExtension(
 
     hasAccessGroups = predicate(Constants.ROLES_CLAIM_URI, SecurityAttributes.ACCESS_GROUPS);
 
+    hasAccessGroupsReadOnly =
+        predicate(Constants.ROLES_CLAIM_URI, SecurityAttributes.ACCESS_GROUPS_READ);
```

Comment:
❗️ If these are supposed to only apply to "read only" cases, then shouldn't we be checking whether or not the action on the KeyValueCollectionPermission is "read"?